src/contexts/v2/projectContext.ts

```diff
@@ -10,6 +10,9 @@ export type V2ProjectContextType = {
   fundingCycle: V2FundingCycle | undefined
   payoutSplits: Split[] | undefined
   reserveTokenSplits: Split[] | undefined
+  tokenAddress: string | undefined
+  terminals: number[] | undefined
```

Comment:
**question** is a terminal a number? 🤔

Reply:
its a string, updated in https://github.com/jbx-protocol/juice-interface/pull/635/commits/1c11f86e958530fdb7d423cd692ffe55af9cd1b0